.github/pull-request-template.md

```diff
@@ -0,0 +1,50 @@
+Fixes #
+
+<!-- Please include the 'why' behind your changes if no issue exists -->
+
+## Proposed Changes
+
+<!-- Please categorize your changes:
+- :gift: Add new feature
+- :bug: Fix bug
+- :broom: Update or clean up current behavior
+- :wastebasket: Remove feature or internal logic
+-->
+
+-
+-
+-
+
+### Pre-review Checklist
+
+<!-- If these boxes are not checked, you will be asked to complete these requirements or explain why they do not apply to your PR. -->
+
+- [ ] **At least 80% unit test coverage**
+- [ ] **E2E tests** for any new behavior
+- [ ] **Docs PR** for any user-facing impact
```

Comment:
Probably doesn't need a separate PR, maybe:

Docs updates for any user-facing changes